mempool/clist_mempool.go

```diff
@@ -662,12 +662,57 @@ func (mem *CListMempool) recheckTxs() {
 	mem.proxyAppConn.FlushAsync()
 }
 
+func (mem *CListMempool) GetNextTransaction(remainBytes int64, remainGas int64, priorBytes []byte) types.Tx {
+	mem.updateMtx.RLock()
+	defer mem.updateMtx.RUnlock()
+	var prior types.Tx
+	prior = priorBytes
```

Comment:
again, prior makes me think of "previous" or "prior knowledge". just use priority